apps/yarn.lock

```diff
@@ -2,13 +2,32 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0":
+"@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/core@^7.4.0":
+"@babel/core@7.1.0":
```

Comment:
It looks like this is still pulling in multiple `@babel/core` versions — I thought it would work with Babel v6, but complain about the peer dependency?